changelog.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Might be cool to add this to our homebrew tap so that we can pull and keep it updated without attaching it to a specific repo, or having to copy paste it around

Reply:
yea.. i also didn't mean to push this! i'll remove